library/Rules/Contains.php

```diff
@@ -13,17 +13,39 @@
 
 namespace Respect\Validation\Rules;
 
-class Contains extends AbstractRule
+/**
+ * @author Alexandre Gomes Gaigalas <alexandre@gaigalas.net>
```

Comment:
Can you add an line before telling what this rule does? You will find that in [Contains.md](https://github.com/Respect/Validation/blob/master/docs/Contains.md)